<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wordle</class>
 <widget class="QMainWindow" name="Wordle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>252</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>252</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wordle Helper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon-192.png</normaloff>:/icon-192.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>37</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Click to focus, type to input letter, delete or backspace to clear; space or tab to skip</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="ClickableLabel" name="label_L1">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Click to focus, type to input letter, delete or backspace to clear; space or tab to skip</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ClickableLabel" name="label_L2">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Click to focus, type to input letter, delete or backspace to clear; space or tab to skip</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="ClickableLabel" name="label_L3">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Click to focus, type to input letter, delete or backspace to clear; space or tab to skip</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="ClickableLabel" name="label_L4">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Click to focus, type to input letter, delete or backspace to clear; space or tab to skip</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="ClickableLabel" name="label_L5">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Click to focus, type to input letter, delete or backspace to clear; space or tab to skip</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>60</y>
      <width>250</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Incorrect Letters</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_NonOccour">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>249</width>
       <height>25</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>Letters not occouring in the word</string>
     </property>
     <property name="whatsThis">
      <string>Letters not occouring in the word</string>
     </property>
     <property name="placeholderText">
      <string>Letters not occouring in the word</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>310</y>
      <width>250</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>125</y>
      <width>252</width>
      <height>182</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_Includes">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Letters Included in the word, pos unknown</string>
       </property>
       <property name="statusTip">
        <string>Letters Included in the word, pos unknown</string>
       </property>
       <property name="whatsThis">
        <string>Letters included in the word, but position unknown</string>
       </property>
       <property name="placeholderText">
        <string>Letters Included in the word, pos unknown</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_NotPos_1">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="statusTip">
        <string>Letters in word, but not on position 1</string>
       </property>
       <property name="whatsThis">
        <string>Letters in word, but not on position 1</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Letters in word, but not on position 1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEdit_NotPos_2">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="statusTip">
        <string>Letters in word, but not on position 2</string>
       </property>
       <property name="whatsThis">
        <string>Letters in word, but not on position 2</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Letters in word, but not on position 2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_NotPos_3">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="statusTip">
        <string>Letters in word, but not on position 3</string>
       </property>
       <property name="whatsThis">
        <string>Letters in word, but not on position 3</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Letters in word, but not on position 3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEdit_NotPos_4">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="statusTip">
        <string>Letters in word, but not on position 4</string>
       </property>
       <property name="whatsThis">
        <string>Letters in word, but not on position 4</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Letters in word, but not on position 4</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="lineEdit_NotPos_5">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="statusTip">
        <string>Letters in word, but not on position 5</string>
       </property>
       <property name="whatsThis">
        <string>Letters in word, but not on position 5</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Letters in word, but not on position 5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>252</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Clear"/>
    <addaction name="separator"/>
    <addaction name="action_Generate"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Clear"/>
   <addaction name="separator"/>
   <addaction name="action_Quit"/>
   <addaction name="separator"/>
   <addaction name="action_Generate"/>
  </widget>
  <action name="action_Clear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Generate">
   <property name="text">
    <string>&amp;Generate</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
